lib/core/public/a11y-check.js

```diff
@@ -16,6 +16,10 @@ axe.a11yCheck = function (context, options, callback) {
 		options = {};
 	}
 
+	if (!('iframes' in options)) {
```

Comment:
We should make sure this is also supported by `axe.run`. Perhaps moving it into run-rules would cover both API methods more easily?